src/Licenses/Entities/License/License.php

```diff
@@ -733,6 +749,7 @@ public function jsonSerialize(): array
             self::COLUMN_MARKETPLACE            => $this->marketplace,
             self::COLUMN_ACTIVE_SEATS           => $this->activeSeats->jsonSerialize(),
             self::COLUMN_FRIENDLY_NAME          => $this->friendlyName,
+            self::COLUMN_VENDOR_BILLING_ID => $this->vendorBillingId,
```

Comment:
I think it is better to pad with space (or either remove padding on every array elements)

Reply:
fixed